pkg/platform/kube/functionres/lazy.go

```diff
@@ -2338,6 +2334,39 @@ func (lc *lazyClient) isPodAutoScaledUp(ctx context.Context, pod v1.Pod) (bool,
 	return false, nil
 }
 
+func (lc *lazyClient) populateDefaultContainerResources(ctx context.Context, container *v1.Container) {
```

Comment:
also, use some UT to make sure population works as expected

Reply:
Added UT in `lazy_test.go`.

Reply:
me > 🙈